packages/web/hooks/use-realtime-sync.ts

```
@@ -35,15 +48,15 @@ const generateUserColor = (userId: string): string => {
     return char.charCodeAt(0) + ((acc << 5) - acc)
   }, 0)
 
-  return colors[Math.abs(hash) % colors.length]
+  return colors[Math.abs(hash) % colors.length] ?? '#FF6B6B'
 }
 
 // Throttle function for performance
 const throttle = (func: Function, delay: number) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)